### PR TITLE
[test](mtmv)Add the cases of upgrade and downgrade of mtmv

### PR DESCRIPTION
**Pull request by zfr9527**
## Proposed changes

[test](mtmv)Add the cases of upgrade and downgrade of mtmv

### CODE REVIEW

**Comment by zfr9527** [comment]
run buildall

File: regression-test/suites/nereids_rules_p0/mv_up_down/load.groovy [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_upgrade_downgrade_prepare_mtmv","p0,mtmv,restart_fe,") {
+
+    String db = context.config.getDbNameByFile(context.file)
+    String orders_tb = "orders"
```
Comment (zddr) [inline regression-test/suites/nereids_rules_p0/mv_up_down/load.groovy:21]: may be repeat with other case?

File: regression-test/suites/nereids_rules_p0/mv_up_down/load.groovy [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        BUILD IMMEDIATE REFRESH AUTO ON MANUAL 
+        partition by(l_shipdate) 
+        DISTRIBUTED BY RANDOM BUCKETS 2 
+        PROPERTIES ('replication_num' = '1')  
+        AS  
+        ${mtmv_sql}
+        """
+
+    def job_name = getJobName(db, mtmv_name)
+    waitingMTMVTaskFinished(job_name)
```
Comment (zddr) [inline regression-test/suites/nereids_rules_p0/mv_up_down/load.groovy:126]: can replace by `waitingMTMVTaskFinishedByMvName(mvName)`,not need `getJobName`

File: regression-test/suites/nereids_rules_p0/mv_up_down/load.groovy (hunk 1 above)
Comment (zfr9527) [reply regression-test/suites/nereids_rules_p0/mv_up_down/load.groovy]: It has been changed to the form of combining the case name and the table name.

File: regression-test/suites/nereids_rules_p0/mv_up_down/load.groovy (hunk 2 above)
Comment (zfr9527) [reply regression-test/suites/nereids_rules_p0/mv_up_down/load.groovy]: It has been modified to the function named "waitingMTMVTaskFinishedByMvName".

**Comment by zfr9527** [comment]
run buildall

**Comment by zfr9527** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by zddr — APPROVED** [review]

**Review by hello-stephen — APPROVED** [review]
LGTM

**Pull request merged**